applications/asset_tracker_v2/src/cloud/azure_iot_hub_integration.c

```diff
@@ -0,0 +1,324 @@
+#include "cloud/cloud_wrapper.h"
+#include <zephyr.h>
+#include <net/azure_iot_hub.h>
+#include <modem/at_cmd.h>
+
+#define MODULE azure_iot_integration
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_CLOUD_INTEGRATION_LOG_LEVEL);
+
+#define AZURE_IOT_HUB_CLIENT_ID_LEN 15
```

Comment:
This needs to be configurable. E.g. a UUIDv4 is 36 bytes.

Reply:
Done.